### ISSUE TITLE
Radio Group with actionable element is not navigable with NVDA browse mode 

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#14042]
Environment Information
Win10
Chrome Version 83.0.4103.116
NVDA 2020.1

Repro steps
NVDA is running in browse mode
1. Open URL
https://fluentsite.z22.web.core.windows.net/components/radio-group/definition
2. Navigate into `Radio Group` example in browse mode
![image](https://user-images.githubusercontent.com/22620150/87543024-41da8e80-c6a4-11ea-8993-48cd579a5a5e.png)

3. Verify you can navigate to the input field in the browse mode

Expected: user is able to navigate to the input next to radio button
Actual: user is not able to navigate to the input 

Comment:
Looks like issue is caused that input is under the element with role="radio"
the input should be place in the HTML next to/after element with role="radio"

**Comment by jurokapsiar** [comment]
caused by https://github.com/microsoft/fluentui/issues/15032
